cli/bpmetadata/markdown_test.go

```diff
@@ -174,3 +174,49 @@ func TestProcessMarkdownContent(t *testing.T) {
 		})
 	}
 }
+
+func TestProcessArchitectureContent(t *testing.T) {
+	tests := []struct {
+		name     string
+		fileName string
+		title    string
+		want     *BlueprintArchitecture
+		wantErr  bool
+	}{
+		{
+			name:     "Architecture details exists as BlueprintArchitecture",
+			fileName: "list-content.md",
+			title:    "Architecture",
+			want: &BlueprintArchitecture{
+				Description: `1. Step 1
+2. Step 2
+3. Step 3`,
```

Comment:
Looking at the example, would it make more sense as a slice of strings?

Reply:
Good idea. Changed.